ui/StatusQ/src/plugin.cpp

```diff
@@ -26,6 +27,10 @@ class StatusQPlugin : public QQmlExtensionPlugin {
         qmlRegisterSingletonType<ModelUtilsInternal>(
             "StatusQ.Internal", 0, 1, "ModelUtils", &ModelUtilsInternal::qmlInstance);
 
+        qmlRegisterSingletonType<PermissionUtilsInternal>("StatusQ.Internal", 0, 1, "PermissionUtils", [](QQmlEngine *, QJSEngine *) {
```

Comment:
As it's totally non-generic functionality it shouldn't be located in `StatusQ`, please at least create a ticket for moving it to status-desktop.

Reply:
https://github.com/status-im/status-desktop/issues/11561